src/Browser/Json.php

```diff
@@ -33,6 +48,63 @@ public function assertMatches(string $expression, $expected): self
         return $this;
     }
 
+    /**
+     * @param string $selector JMESPath selector
+     */
+    public function assertHas(string $selector): self
+    {
+        Assert::try(fn() => $this->search("length({$selector})"));
+
+        Assert::pass();
```

Comment:
Just released `zenstruck/assert` `1.1.0`. Let's bump in composer.json to take advantage of https://github.com/zenstruck/assert/pull/13.